packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -3172,60 +3173,78 @@ Content-Type: application/octet-stream
 --abcde12345--
 ```
 
-### Payload_MultiPart_FormData_fileWithHttpPartOptionalContentType
+### Payload_MultiPart_FormData_HttpParts_NonString_float
 
-- Endpoint: `post /multipart/form-data/file-with-http-part-optional-content-type`
+- Endpoint: `post /multipart/form-data/non-string-float`
 
-Please send request twice, first time with no content-type and second time with content-type "application/octet-stream". Expect request:
+Expect request:
 
 ```
 POST /upload HTTP/1.1
 Content-Length: 428
 Content-Type: multipart/form-data; boundary=abcde12345
 
 --abcde12345
-Content-Disposition: form-data; name="profileImage"; filename="<any-name-is-ok>"
-Content-Type: application/octet-stream
+Content-Disposition: form-data; name="temperature"
+Content-Type: text/plain
 
-{…file content of .jpg file…}
+0.5
 --abcde12345
 ```
 
-### Payload_MultiPart_FormData_fileWithHttpPartRequiredContentType
+### Payload_MultiPart_FormData_jsonArrayAndFileArrayAndOthers
 
-- Endpoint: `post /multipart/form-data/check-filename-and-required-content-type-with-httppart`
+- Endpoint: `post /multipart/form-data/complex-parts`
```

Comment:
nit: change sth like: form-data/json-array-and-file-array-and-others

Reply:
It doesn't matter so will keep it